src/test/java/productHistoricalTest/AddCustomerTest.java

```diff
@@ -41,7 +43,8 @@ public void userSignin() throws Exception {
 			addCustomerPage.selectCurrency(rowNum);
 			addCustomerPage.navigateBottom();
 			addCustomerPage.clickSaveChangesButton();
-			addCustomerPage.verifyConfirmationMsg("Saved Customer " + (rowNum+1)  + "Successfully For Parent Company");
+			String name= driver.findElement(By.xpath("//tr[@class='active']//strong//span")).getText();
+			addCustomerPage.verifyConfirmationMsg("Saved " + name + " Successfully For Parent Company");
```

Comment:
@mayankchhajer There is no need to pass the name in the message, So Please remove the String name variable.